sale_commission_formula/readme/USAGE.rst

```diff
@@ -1,3 +1,38 @@
 To use this module, you need to:
 
-* Go to Sales, Commission Types and create a commission with type formula
+Prerequisite: https://github.com/OCA/commission/blob/14.0/sale_commission/readme/CONFIGURE.rst
```

Comment:
This paragraph should be in INSTALL.rst and you should avoid absolute references, as on next versions, the version change may be forgotten and it will be misleading.